app/src/test/java/com/krivosheev/mikhail/unittests/RandomNameGeneratorImplTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.krivosheev.mikhail.unittests
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Assertions.assertNotEquals
+import org.junit.jupiter.api.Test
+
+class RandomNameGeneratorImplTest {
+
+    private val matcher = Regex("[a-z]{8}")
+    private val randomNameGenerator = RandomNameGeneratorImpl()
+
+    @Test
+    fun `should match alphabetical characters when generating random name for every call`() {
+        val actual = randomNameGenerator.generateRandomName()
+
+        assertTrue(actual.matches(matcher))
+    }
+
+    @Test
+    fun `should match length of the generated names when generating random name for every call`() {
```

Comment:
As you use pattern ```[a-z]{8}``` it already check the length. Please check it and remove this method if my assumption is correct.